<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>541</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 174);
background-color: rgb(170, 170, 255);</string>
  </property>
  <widget class="QWidget" name="centalwidget">
   <widget class="QLabel" name="label_accesorio">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aharoni</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>DETALLE DE ACCESORIOS</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>MASCOTA</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ACCESORIO</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="entrada_mascota_accesorio">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>100</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="entrada_accesorios">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>140</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>180</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TALLA</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="entrada_talla_accesorio">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>180</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>210</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TIPO</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="entrada_tipo_accesorio">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>210</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>250</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>PRECIO</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="entrada_precio_accesorio">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>250</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkbox_ofertas">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>300</y>
      <width>271</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Recibir ofertas por mail</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_cliente">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>350</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 247);</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>si</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>no</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>350</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>¿Ya eres cliente?</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_standar">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>390</y>
      <width>82</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Standar</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>420</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Tipo de envío </string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_urgente">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>420</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Urgente</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_recogida">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>450</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Recogida</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>90</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>IMAGEN</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_imagen">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>150</y>
      <width>241</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
